tests/src/test/scala/whisk/core/cli/test/WskCliBasicUsageTests.scala

```diff
@@ -555,7 +558,9 @@ class WskCliBasicUsageTests extends TestHelpers with WskTestHelpers {
     withActivation(wsk.activation, run) { activation =>
       activation.response.status shouldBe "success"
       val fields = activation.response.result.get.convertTo[Map[String, String]]
-      fields("api_host") shouldBe WhiskProperties.getApiHostForAction
+      if (!noApiHostCheck) {
```

Comment:
@drcariel - nit.. double negative is a littler harder to read that `if (apiHostCheck) {`